BenchmarkComponent/BenchmarkComponent.cpp

```diff
@@ -61,4 +65,63 @@ namespace winrt::BenchmarkComponent::implementation
     void ClassWithMultipleInterfaces::DefaultDoubleProperty(double val)
     {
     }
+
+    ClassWithMarshalingRoutines::ClassWithMarshalingRoutines()
+    {
+    }
+
+    hstring ClassWithMarshalingRoutines::DefaultStringProperty()
+    {
+        return hstring();
+    }
+
+    void ClassWithMarshalingRoutines::DefaultStringProperty(hstring val)
+    {
+    }
+    
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultKeyValuePairProperty()
+    {
+        Windows::Foundation::Collections::ValueSet valueSet;
+        valueSet.Insert(hstring(L"key"), PropertyValue::CreateString(L"value"));
+        return valueSet.First().Current();
+    }
+
+    void ClassWithMarshalingRoutines::DefaultKeyValuePairProperty(Windows::Foundation::IInspectable val)
+    {
+    }
+
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultNullableProperty()
+    {
+        return IReference<INT32>(123);
+    }
+
+    void ClassWithMarshalingRoutines::DefaultNullableProperty(Windows::Foundation::IInspectable val)
+    {
+    }
+
+    Windows::Foundation::IInspectable ClassWithMarshalingRoutines::DefaultArrayProperty()
+    {
+        int values[]{ 0, 42, 1729, -1 };

```

Comment:
Should we move most of the construction logic to the constructor as the purpose isn't to benchmark that but rather the interop to the consuming projection.